app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentEvents.kt

```diff
@@ -0,0 +1,23 @@
+package org.simple.clinic.scheduleappointment
+
+import org.simple.clinic.widgets.UiEvent
+import org.threeten.bp.temporal.ChronoUnit
+import java.util.UUID
+
+data class SheetCreated(val initialIndex: Int, val patientUuid: UUID) : UiEvent
```

Comment:
Considering that it has values used only in scheduling of appointments, can you rename this event to make it specific?